recipe/kernels/xcpp17-Clad/kernel.json.in

```diff
@@ -0,0 +1,11 @@
+{
+  "display_name": "C++17-Clad”,
+  "argv": [
+      "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_BINDIR@/xcpp",
+      "-f",
+      "{connection_file}",
+      "-fplugin=$CONDA_PREFIX/lib/clad.dylib",
```

Comment:
Does this work? I would expect `@CMAKE_INSTALL_PREFIX@`, `@CMAKE_INSTALL_BINDIR@` and `$CONDA_PREFIX` to  need manual replacement as part of the build script.

Reply:
I think we also need `"-I /path/to/include"`

Reply:
I think `$PREFIX/include` is included by default (conda is basically a full sysroot that replaces the host's `/include`) so it should be fine as-is. Or are there additional include paths that are needed?

Reply:
Yes, it's included by default

Reply:
Nice, thanks!